pkg/api/types.go

```diff
@@ -1035,6 +1039,26 @@ type Service struct {
 	Status ServiceStatus `json:"status,omitempty"`
 }
 
+// ServiceAccount binds together:
+// * a name, understood by users, and perhaps by peripheral systems, for an identity
+// * a principal that can be authenticated and authorized
```

Comment:
That was copied/pasted from docs/design/service_accounts.md, and could probably use some clarification/rewording. Traditionally, a principal is what you reference in policy rules. Principals can include include human users, machine accounts, etc... I think it's an attempt to avoid "user", since that implies a person.